pkgs/development/tools/xcode-install/default.nix

```
@@ -0,0 +1,15 @@
+{ bundlerApp }:
```

```suggestion
{ lib, bundlerApp }:
```

`lib` is not in scope.

---

Ah, you're right, didn't change this after adding the metadata. Whoops!